packages/emotion/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@leafygreen-ui/emotion",
+  "version": "0.9.0",
+  "description": "leafyGreen UI Kit Emotion Library",
```

Comment:
Maybe "LeafyGreen UI Kit's custom instance of Emotion"

Reply:
👍